petastorm/fs_utils.py

```diff
@@ -211,3 +212,15 @@ def normalize_dir_url(dataset_url):
     dataset_url = dataset_url[:-1] if dataset_url[-1] == '/' else dataset_url
     logger.debug('directory url: %s', dataset_url)
     return dataset_url
+
+def update_storage_options(options, inherited=None):
+    if not inherited:
```

Comment:
Please add a docstring and a unit test.

Reply:
Removed this method. Using the same method from fsspec library.